charts/openfga/templates/deployment.yaml

```diff
@@ -34,17 +34,6 @@ spec:
       serviceAccountName: {{ include "openfga.serviceAccountName" . }}
       securityContext:
         {{- toYaml .Values.podSecurityContext | nindent 8 }}
-      {{- if and (has .Values.datastore.engine (list "postgres" "mysql")) .Values.datastore.applyMigrations }}
```

Comment:
I understand the other changes in this PR, but I don't understand why you dropped this. Could you elaborate on why you removed this?

Reply:
pinging @jbouchery :)